blocks/footer/footer.css

```
@@ -16,5 +17,134 @@ footer .footer p {
 
 footer .footer-slogan {
   font-size: var(--footer-font-size-slogan);
+  padding-bottom: 40px;
+  font-style: italic;
+}
+
+footer li {
+  display: inline-block;
+  margin-right: 1rem;
+}
+
+footer a {
+  color: #fff;
+}
+
+footer li a:any-link {
+  color: white;
+}
+
+.footer-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.footer-container .right-side .icon{
+  fill: #949494;
+  transition: all 0.3s ease;
+  line-height: 1.6;
+  width: 32px;
+  height: 32px;
+  margin-bottom: 19px;
+}
+
+.footer-container [class^="footer-menu-list-"]{
+    padding-left:0;
+}
+
+.footer-container ul.footer-menu-list-0 li a {
+  font-size: 14px;
+  letter-spacing: 3px;
+}
+
+.footer-container ul.footer-menu-list-1 li a {
+  font-size: 14px;
+  opacity: 0.5;
+}
+
+
+.footer-container .right-side > p:nth-child(7) > a {
+  opacity: 0.5; 
+  color: #fff;
+ }
+
+.footer-container ul.footer-menu-list-1 li a:hover {
+  opacity: 1;
+}
+
+.footer-container .middle-div .trademark-container .trade-mark a {
+  color: #fff;
+  opacity: 0.5;
+}
+
+.footer-container .right-side > p:nth-child(7) > a:hover {
+  opacity: 1;
+ }
+
+.footer-container .middle-div .trademark-container .trade-mark a:hover {
+  opacity: 1;
+}
+
+.footer-container .right-side {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+.footer-container .right-side img {
+  width: 200px;
+  height: auto;
+}
+
+ .trademark-container{
+  display: flex;
+  justify-content: center;
+  height: 100px;
+  align-items: flex-end;
+ }
+
+@media (max-width: 991px) {
+  footer .footer {
+    padding-left: 15px;
+    padding-right: 15px;
+  }
+
+  footer hr{
+    display: none;
+  }
+}
+
+.top-btn-container{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 50px;
+}
+
+.top-btn {
```

I added the footer prefix. And I made it more specific, which makes it override the default `a:any-link` styles. 
```suggestion
footer a.button.jump-to-top:any-link {
```

---

I see making it more specific so css is scoped to footer only